src/Cli/dotnet/commands/dotnet-workload/InstallType.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
This file probably belongs alongside the other common install code. So far I've been putting all of it under dotnet-workload/install, but if we want to move it to another common dir, that works too (also doesn't have to be done in this PR).

Reply:
I'll probably leave it where it is for now until I get the rest in place and see what it looks like. I might very well end up with a whole separate Windows folder